### DISCUSSION TITLE
StashApp on QNAP Installation Failure: Container Station cannot find docker-compose.yml path

### DISCUSSION THREAD

**Discussion by AyugeFan** [stashapp/stash#5077] [Q&A]
I'm getting on error with my docker configuration. I"m using Container Station on a QNAP TVS-871. 

The only thing I've changed is  the path for the docker compose file. During setup, from the advanced setting menu, I went into the environment tab and joint the STASH_CONFIG_FILE variable to the location of my docker-compose.yml file I downloaded.

It returns this error when I start the container.

Any advice would be appreciated.
Thank you,

____

ERRO[2024-07-16 12:03:43] config initialization error: could not write to provided config path "/share/CACHEDEV2_DATA/StashAppData/docker-compose.yml": open /share/CACHEDEV2_DATA/StashAppData/docker-compose.yml: no such file or directory

<img width="532" alt="Screenshot 2024-07-16 at 8 02 36 AM" src="https://github.com/user-attachments/assets/3f23c503-8b0f-4c4a-bcce-5242a43b5e25">

**Comment by everforyou** [comment]
Make sure the environment variable is set like this:
```bash
Variable: STASH_CONFIG_FILE
Value: /share/CACHEDEV2_DATA/StashAppData/docker-compose.yml
```
